js/jsmind.js

```diff
@@ -1666,6 +1669,31 @@
             for (var i = 0; i < l; i++) {
                 this.event_handles[i](type, data);
             }
+        },
+
+        // Drag the whole mind map with your mouse, when it's larger that the container
+        _drag_nodes: function () {
+            if (this.options.draggable) {
+                // Avoid scrollbars when mind map is larger than the container (e_panel = jsmind-inner)
+                this.view.e_panel.style = 'overflow: hidden'
```

Comment:
It's better to keep the scrollbars in this case.
OR
I prefer to use another setting field to control whether the scrollbars should be shown or not